bigbluebutton-web/grails-app/conf/bigbluebutton.properties

```diff
@@ -54,7 +54,12 @@ BLANK_SVG=/usr/share/bigbluebutton/blank/blank-svg.svg
 #----------------------------------------------------
 # Number of minutes the conversion should take. If it takes
 # more than this time, cancel the conversion process.
-maxConversionTime=20
+maxConversionTime=5
+
+# Number of milliseconds the conversion should take per page. If the total
+# conversion time exceeds maxConversionTime x number of pages then the
+# conversion will be cancelled.
+maxPageConversionTime=30000
```

Comment:
I suggest using seconds here instead of milliseconds.
We could name it `maxPageConversionTimeInSeconds`, following the pattern we've been using in recent configs.
Most of them use `InMinutes`, but in this case, seconds make more sense since we might want to allow a short duration.